docker/install/ubuntu_install_tflite.sh

```diff
@@ -26,12 +26,18 @@ cd flatbuffers
 cmake -G "Unix Makefiles" -DCMAKE_BUILD_TYPE=Release
 make install -j8
 cd ..
-rm -rf flatbuffers
 
 # Install flatbuffers python packages.
 pip3 install flatbuffers
 pip2 install flatbuffers
 
+# Build the TFLite static library, necessary for building with TFLite ON.
+# The library is built at:
+# tensorflow/tensorflow/lite/tools/make/gen/*/lib/libtensorflow-lite.a.
+git clone https://github.com/tensorflow/tensorflow --branch=r2.0
```

Comment:
why not 2.1.0 which is the latest stable version and what we install in the tree. I don’t think it’s worth the version skew.


Ramana

Reply:
@u99127 Good catch, thank you. I noticed your PR updating the version yesterday but I forgot I was specifying 2.0 here. Will update :)